torchvision/datasets/cifar.py

```diff
@@ -112,9 +112,10 @@ def __getitem__(self, index: int) -> Tuple[Any, Any]:
         """
         img, target = self.data[index], self.targets[index]
 
-        # doing this so that it is consistent with all other datasets
-        # to return a PIL Image
-        img = Image.fromarray(img)
+        if self.get_image_backend() == 'PIL':
```

Comment:
This assumes that we only have two backends, right? What if the user had specified `accimage`?